src/sources/azure_blob/queue.rs

```diff
@@ -194,7 +253,7 @@ async fn proccess_event_grid_message(
                     Err(e) => {
                         // Logging as trace, as it will be retired.
                         trace!("Failed to get response: {}", e);
-                        return None;
+                        return Err(ProcessingError::FailedToGetBlob {error: e});
```

Comment:
IIUC, all errors in this function are logged in the call site, so this trace and a few others are not necessary maybe?

Reply:
Actually yes, I just kept them as some are traces but I log them in emit with error level. If that is not big difference will clean those